src/lib/css-grammar-parser.js

```diff
@@ -37,6 +37,8 @@ const primitives = new Map([
   ["position", {url: "https://drafts.csswg.org/css-values-4/#typedef-position"}]
 ]);
 
+const numericTypes = ["integer", "number", "percentage", "number-percentage", "length-percentage", "frequency-percentage", "angle-percentage", "time-percentage", "dimension", "length", "angle", "time", "frequency", "resolution"];
```

Comment:
What is this definition for? It does not seem to be used anywhere else in the code.

Reply:
I've now removed it - I had initially thought of throwing when range was used on a non-numeric type, but that goes too close to a validator (vs a parser).